src/_guides/language/sound-problems.md

```diff
@@ -416,74 +414,69 @@ HoneyBadger(Eats food, String name)
 <hr>
 
 <a name="uses-dynamic-as-bottom"></a>
-### The function expression type ... isn't of type ...
+### The argument type ... can't be assigned to the parameter type ...
 
-{% comment %}
-<!-- code-excerpt "strong/analyzer-results-stable.txt" retain="/The function expression type.*common_problems/" replace="/'bool.*?\)'/'...'/g" -->
+<?code-excerpt "analyzer-results-stable.txt" retain="/The argument type.*bool Function/" replace="/'bool.*?\)'/'...'/g; /-(.*?):(.*?):(.*?)-/-/g"?>
 ```nocode
-[PENDING: fix this]
+error - The argument type '...' can't be assigned to the parameter type '...'. - argument_type_not_assignable
 ```
-{% endcomment %}
 
 In Dart 1.x `dynamic` was both a [top type][] (supertype of all types) and a
```

Comment:
This is one section I wasn't sure how to best include/explain this without mentioning Dart 1. I'm open to keeping it as is, but if you have any suggestions here, that'd be great too. Thanks!